venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -35,51 +35,223 @@ def process(client, note, invitation):
+                    'description': 'Please carefully review the template below before you click submit to send out recruitment emails. Make sure not to remove the parenthesized tokens.',
+                    'order': 5,
+                    'required': True,
+                    'default': recruitment_email_body
+                }
+            }
+        },
+        signatures = [conference.get_program_chairs_id()]
+    ))
+    else:
+        recruitment_invitation = client.post_invitation(openreview.Invitation(
+        id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Recruitment',
+        super = 'OpenReview.net/Support/-/Recruitment',
+        invitees = readers,
+        reply = {
+            'forum': forum.id,
+            'replyto': forum.id,
+            'readers' : {
+                'description': 'The users who will be allowed to read the above content.',
+                'values' : readers
+            },
+            'writers' : {
+                'description' : 'The users who will be allowed to edit the above content',
+                'values' : [conference.get_program_chairs_id(), 'OpenReview.net/Support']
+            },
+            'content': {
+                'title': {
+                    'value': 'Recruitment',
+                    'required': True,
+                    'order': 1
+                },
+                'invitee_role': {
+                    'description': 'Please select the role of the invitees in the venue.',
+                    'value-radio': ['reviewer'],
+                    'default': 'reviewer',
+                    'required': True,
+                    'order': 2
+                },
+                'invitee_details': {
+                    'value-regex': '[\\S\\s]{1,50000}',
+                    'description': 'Please provide line separated invitee details where each line has the pair - email,name. E.g. captain_rogers@marvel.com, Captain America',
+                    'required': True,
+                    'order': 3
+                },
+                'invitation_email_subject': {
+                    'value-regex': '.*',
+                    'description': 'Please carefully review the email subject for the recruitment emails. Make sure not to remove the parenthesized tokens.',
+                    'order': 4,
+                    'required': True,
+                    'default': recruitment_email_subject
+                },
+                'invitation_email_content': {
+                    'value-regex': '[\\S\\s]{1,10000}',
+                    'description': 'Please carefully review the template below before you click submit to send out recruitment emails. Make sure not to remove the parenthesized tokens.',
+                    'order': 5,
+                    'required': True,
+                    'default': recruitment_email_body
+                }
+            }
+        },
+        signatures = [conference.get_program_chairs_id()]
```

Comment:
Shouldn't it be readers so Support can run this too? Same for the invites below.

Reply:
Support can still run these invitations with Program Chair permissions.